### PR TITLE
Jd 2024 06 fix email translation

### PR DESCRIPTION
**Pull request by goapunk**
**Tasks**
- [ ] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

File: changelog/2721.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- add missing template for password_set view
```
Comment (m4ra) [inline changelog/2721.md:3]: did u forget to add the template in the PR?

**Review by m4ra — CHANGES_REQUESTED** [review]

File: changelog/2721.md (hunk 1 above)
Comment (goapunk) [reply changelog/2721.md:3]: ah :exploding_head: and now it's probably gone. Will check

Comment (goapunk) [reply changelog/2721.md:3]: ah @m4ra the template is already merged, I just forgot the changelog in that PR so I added it here, sorry

**Review by m4ra — APPROVED** [review]
thank you! 
did you push to transifex? I am going to merge this, and we make a new PR when translations are done.

**Comment by goapunk** [comment]
> thank you! did you push to transifex? I am going to merge this, and we make a new PR when translations are done.

no I haven't pushed it

**Comment by m4ra** [comment]
Can you push to transifex and ping Tietje

**Pull request merged**